### ISSUE TITLE
withCustomActions decorator overides  not allow to use custom methods

### ISSUE THREAD

**Issue by romgere** [Naeka/ember-custom-actions#1]
Hi,

I tried to switch from https://github.com/Exelord/ember-custom-actions to this add-on & had some trouble with the decorator.

The decorators always redefine custom actions methods & does not allow to customize these methods on the class using the decorators...

So I end up by using the decorator in the application adapter to allow child adapter to overrides custom actions methods...

When reading, the doc, it does not seems to be the behavior wanted 🤔 

Maybe the decorator should replace the class methods only when they are `undefined` to prevent this ?

Just for information the code I use : 

```js
import ApplicationAdapter from './application'
import { withCustomActions } from '@naeka/ember-custom-actions'

@withCustomActions
export default class RuleBaseAdapter extends ApplicationAdapter {
  urlForCustomAction(modelName, id, snapshot, actionId) {
    // handling some custom url here
  }
}
```

```js
import Model from '@ember-data/model'
import { customAction } from '@naeka/ember-custom-actions'

export default class RuleBaseModel extends Model {
  ...
  updateCategory = customAction('updateCategory')
}
```

& when calling `model.updateCategory()` it never call the `adapter.urlForCustomAction` but call the default method created in the decorator ([here](https://github.com/naeka/ember-custom-actions/blob/master/addon/decorators/adapter.js#L14))

**Comment by tomdrn** [comment]
Hi !

This is not the expected behavior. While waiting for a fix here's a quick workaround :

```javascript
import ApplicationAdapter from './application'
import { withCustomActions } from '@naeka/ember-custom-actions'

@withCustomActions
class CustomActionsApplicationAdapter extends ApplicationAdapter {};

export default class RuleBaseAdapter extends CustomActionsApplicationAdapter {
    urlForCustomAction(modelName, id, snapshot, actionId) {
       // handling some custom url here
    }
}
```

**Comment by tomdrn** [comment]
@romgere Thanks for reporting ! The bug has been fixed in release 0.1.1
